packages/settings-library/src/settings_library/tracing.py

```diff
@@ -15,3 +15,7 @@ class TracingSettings(BaseCustomSettings):
     TRACING_OPENTELEMETRY_COLLECTOR_PORT: Annotated[
         RegisteredPortInt, Field(description="Opentelemetry compatible collector port")
     ]
+    TRACING_SAMPLING_PROBABILITY: Annotated[
```

Comment:
this is required so you shall create the MR according to the ENV workflow (which I think is linked in the PR template)